### PR TITLE
Implement Popup windows

### PR DESCRIPTION
**Pull request by mrjuda**
Generate portfolio cards' popup windows dynamically using Javascript.

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -665,6 +809,21 @@ form .cta-button:hover {
...
+
+  .popup-container {
+    width: 90%;
+    justify-self: center;
+    margin-top: -11rem;
+  }
+
+  .popup-card {
+    justify-self: center;
+  }
```
Comment (Amrhub) [inline styles/main.css:823]: - [x] Let's make sure our image in the pop-up is the same size in desktop view to design to make your project more appealing and get read of `negative space`

Your implementation | Figma Design
:------:|:-----:
image| image

File: styles/main.css [hunk 2]
```diff
@@ -272,8 +377,13 @@ form p {
   padding-bottom: 50px;
 }
 
+.dynamics {
+  display: grid;
+}
+
 .card {
   display: flex;
+  justify-self: center;
```
Comment (Amrhub) [inline styles/main.css:386]: - [ ] [OPTIONAL] Our cards display now is somehow changed in desktop view and acting like the mobile version, Kindly remainder to fix it, It's content and image are not side by side as it was before in your latest commits and currently as you can see in the photo below, project image takes up big space I couldn't fit whole card in a single screenshot

Your implementation | Figma Design
:------:|:-----:
image | image

**Review by Amrhub — CHANGES_REQUESTED** [review]
Hi @mrjuda,  👋🏻 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!
These changes aim to add some awesomeness to your awesome project and code skills.

I really feel you. I wish you get past these time if you want to text me on slack for any help don't hesitate @mrjuda 

## To Highlight ✔️ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles/main.css (hunk 2 above)
Comment (mrjuda) [reply styles/main.css:386]: I'm so sorry for not fixing this issue right now. But I still need to finish the other 3 project submissions to the reviewers. I'll fix it, as soon as possible.

File: styles/main.css (hunk 1 above)
Comment (mrjuda) [reply styles/main.css:823]: Done, thank you!

**Review by gemmen29 — APPROVED** [review]
Hi @mrjuda, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**